gclientgen/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/googleapis/google-cloud-rust/gclientgen
```

Comment:
What about just `github.com/googleapis/google-cloud-rust/generator` for the name?

Reply:
Done.